src/wrappers/PhylorefWrapper.js

```diff
@@ -643,48 +660,108 @@ class PhylorefWrapper {
     if (!has(phylorefAsJSONLD, '@id')) phylorefAsJSONLD['@id'] = fallbackIRI;
     phylorefAsJSONLD['@type'] = 'owl:Class';
 
-    // All phyloreferences are subclasses of phyloref:Phyloreference.
-    phylorefAsJSONLD.subClassOf = 'phyloref:Phyloreference';
-
-    // Construct an equivalentClass expression for this phyloreference.
+    // Construct a class expression for this phyloreference.
     const internalSpecifiers = phylorefAsJSONLD.internalSpecifiers || [];
     const externalSpecifiers = phylorefAsJSONLD.externalSpecifiers || [];
 
-    // We might need to make additional JSON-LD.
-    // So we reset our additional class counts and records.
-    PhylorefWrapper.additionalClassCount = 0;
-    PhylorefWrapper.additionalClassesByLabel = {};
-    phylorefAsJSONLD.hasAdditionalClass = [];
+    // In the following section: we ran into a bug (https://github.com/phyloref/phyx.js/issues/57)
+    // that was caused by OWL reasoners deciding that two phylorefs were identical
+    // because they shared an equivalentClass expression. To prevent this from
+    // happening, we now set up each phyloreference as a subclass of every logical
```

Comment:
Small nitpick: I don't think it helps someone to understand _how_ we got here. The comment just needs to say _why_ we're doing this.

Reply:
Fixed in 097dc458ab65.